HPXMLtoOpenStudio/resources/schedules.rb

```diff
@@ -329,78 +333,88 @@ def createSchedule()
     prev_wkdy_rule = nil
     prev_wknd_vals = nil
     prev_wknd_rule = nil
-    for m in 1..12
-      date_s = OpenStudio::Date::fromDayOfYear(day_startm[m], assumedYear)
-      date_e = OpenStudio::Date::fromDayOfYear(day_endm[m], assumedYear)
+    periods = []
+    if @begin_month < @end_month
```

Comment:
What if you specified a run period that was contained within a single calendar month? For example, 1/1 to 1/31. Then you'd have more than one period?

Reply:
Thanks for catching.  If a user specifies a run period of a single calendar month, the user should get one period.